config/database.yml

```
@@ -3,6 +3,9 @@ default: &default
   encoding: unicode
   pool: 5
   timeout: 5000
+  host: <%= ENV['POSTGRES_HOST'] || '127.0.0.1' %>
+  username: <%= ENV['POSTGRES_USER'] || 'root' %>
+  password: <%= ENV['POSTGRES_PASSWORD'] || 'password' %>
```

Just a small suggestion:

```suggestion
  host: <%= ENV.fetch('POSTGRES_HOST', '127.0.0.1') %>
  username: <%= ENV.fetch('POSTGRES_USER', 'root') %>
  password: <%= ENV.fetch('POSTGRES_PASSWORD', 'password') %>
```